src/apiary.apib

```diff
@@ -1544,6 +1544,61 @@ level `conversions` array.
                 + conversion (Conversion)
                     + conversion_type: 3 (number)
 
+# Group Alert
+
+Alerts are used across application to notify users (when subscribed) of events happening in platform.
+
+## Alerts [/alerts/alert]
+
+### Get All Available Alerts [GET]
++ Response 200 (application/json; charset=utf-8)
+    + Attributes
+        + alerts (array)
+            + (Alert Base)
+                + id (number)
+
+# Notifiers [/alerts/notifier]
+
+## Get All Available Notifiers [GET]
+
++ Response 200 (application/json; charset=utf-8)
+    + Attributes
+        + notifiers (array)
+            + (Notifier Base)
+                + id (number)
+
+## Notifications [/alerts/notification]
+
+### List User Notifications [GET]
++ Response 200 (application/json; charset=utf-8)
+    + Attributes
+        + notifications (array)
+            + (Notification Base)
+                + id (number)
+
+### Delete User Notifications [DELETE]
++ Request (application/json; charset=utf-8)
+    + Attributes
+        + notifications (array)
+            + (object)
+                + id (number)
+
++ Response 200
+
+### Create New Notification [POST]
++ Request (application/json; charset=utf-8)
+    + Attributes
+        + notification (object)
+            + alert_id (number)
+            + user_id (number)
+            + notifier (object)
+                + notifier_id (number)
+                + values (object) - data specific to a notifier
+
++ Response 200 (created)
```

Comment:
This is invalid syntax, (created) and (bad request) will render as a content type.